packages/front-end/components/Features/SDKConnections/SDKConnectionsList.tsx

```diff
@@ -40,17 +56,109 @@ export default function SDKConnectionsList() {
     project
   );
 
+  const useNewEmptyStateLayout = useFeatureIsOn(
+    "sdk-connections-new-empty-state"
+  );
```

Comment:
Just realized we should change how we're evaluating this feature flag.  We only want to check the feature flag if the user is going to see the empty state.  If someone already has SDK connections, we don't want to include them in the experiment since it will just add noise.

We did something similar for the onboarding test we ran:

```ts
const gb = useGrowthBook();

let useNewEmptyStateLayout = false;
if (connections.length === 0 && canCreateSDKConnections) {
  useNewEmptyStateLayout = gb.isOn("sdk-connections-new-empty-state");
}
```

Reply:
Oh that's a good catch! Updated following your recommendation.